db/migrate/20170815175201_swap_column_names.rb

```diff
@@ -0,0 +1,7 @@
+class SwapColumnNames < ActiveRecord::Migration
+  def change
+  	rename_column :scheduled_items, :admission, :admission_temp
+  	rename_column :scheduled_items, :audience, :admission
+  	rename_column :scheduled_items, :admission_temp, :audience
+  end
```

Comment:
That question doesn't seem relevant here.

Also, if you do research a non-obvious issue, cite your reference. Even if you're just working alone, there are times you'll thank yourself later.